src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/personalrelationships/PrisonerRestrictionsReconciliationService.kt

```diff
@@ -14,18 +14,22 @@ import uk.gov.justice.digital.hmpps.prisonertonomisupdate.services.awaitBoth
 
 @OptIn(ExperimentalCoroutinesApi::class)
 @Service
-class PrisonerReconciliationReconciliationService(
+class PrisonerRestrictionsReconciliationService(
   private val telemetryClient: TelemetryClient,
   private val nomisApiService: ContactPersonNomisApiService,
-  @Value("\${reports.contact-person.prisoner-restrictions.reconciliation.page-size:10}") private val pageSize: Int = 10,
+  private val mappingApiService: ContactPersonMappingApiService,
+  private val dpsApiService: ContactPersonDpsApiService,
+
+  @param:Value($$"${reports.contact-person.prisoner-restrictions.reconciliation.page-size:10}") private val pageSize: Int = 10,
 ) {
```

Comment:
interesting IntelliJ suggestion; I suppose you can escape using `$$` in Kotlin these days